tests/e2e/specs/dashboard/documentTitle.js

```diff
@@ -33,6 +33,7 @@ describe('Document Title', () => {
     await expect(dashboardNavigation).toClick('a', {
       text: 'Explore Templates',
     });
+    await page.waitForTimeout(100);
```

Comment:
Why are these timeouts here needed? What happens during that time?

Reply:
You are clicking on a link and waiting for the page to render. tried await navigation but it didn't. It is just a small delay to see the page rendered and the title changes.

Reply:
Yeah awaiting navigation does not work because there is none — it's a single page app.

I'm just wary of adding arbitrary timeouts everywhere, that's all.

Reply:
I tried await navigation because that hash does change, but I didn't think it would work. 

Again, some of these fixes here are hacks, as the e2e run slower in GHA than locally.